src/components/toolbox/Chart/chart.stories.js

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import StoryWrapper from '../../../../.storybook/components/StoryWrapper/StoryWrapper';
+import Box from '../box';
+import BoxHeader from '../box/header';
+import Chart from './index';
+import { styles } from '../../../utils/chartOptions';
+import style from './chart.stories.css';
+
+storiesOf('Toolbox', module)
+  .add('Chart', () => (
+    <StoryWrapper>
+      <Box className={style.wrapper}>
+        <BoxHeader><h1>Line Chart</h1></BoxHeader>
+        <div className={style.container}>
+          <Chart
+            type="line"
+            data={{
+              labels: ['Jan', 'Feb', 'Mar', 'Apr', 'May', 'Jun', 'Jul', 'Oug', 'Sep', 'Oct', 'Nov', 'Dec'],
+              datasets: [
+                {
+                  label: '123456789L',
+                  data: [35, 55, 23, 67, 98, 46, 134, 66, 70, 33, 100, 120],
+                  backgroundColor: styles.whiteSmoke,
+                  borderColor: styles.borderColor,
+                  pointBorderColor: styles.borderColor,
+                  pointBackgroundColor: styles.whiteColor,
+                  pointHoverBackgroundColor: styles.whiteColor,
+                  pointHoverBorderColor: styles.ultramarineBlue,
+                  pointHoverBorderWidth: 4,
+                  borderWidth: 2,
+                },
+              ],
+            }}
+          />
+        </div>
+      </Box>
+
+      <Box className={style.wrapper}>
+        <BoxHeader><h1>Bar Chart</h1></BoxHeader>
+        <div className={style.container}>
+          <Chart
+            type="bar"
+            data={{
+              labels: ['Jan', 'Feb', 'Mar', 'Apr', 'May', 'Jun', 'Jul', 'Oug', 'Sep', 'Oct', 'Nov', 'Dec'],
+              datasets: [
+                {
+                  label: '123456789L',
+                  data: [35, 55, 23, 67, 98, 46, 134, 66, 70, 33, 100, 120],
+                  barPercentage: 1,
+                  categoryPercentage: 0.5,
+                  barThickness: 'flex',
+                  maxBarThickness: 14,
+                  minBarLength: 0,
```

Comment:
Similar comment to the above. Removing styles makes it look like this:
![Screenshot 2019-11-18 at 08 58 15](https://user-images.githubusercontent.com/1254342/69034715-3edab180-09e2-11ea-86cc-b17bc3f3803c.png)